src/callback.jl

```diff
@@ -4,23 +4,22 @@
 Print diagnostic information in each assembly iterations.
 """
 function print_callback(info)
```

Comment:
This callback is never executed in the tests. Do this somewhere to avoid making a stupid mistake without notice.

Reply:
Yes that's a good point. I will do a dedicated test for that.